versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.9.1"
```

Comment:
Minor: I'd be tempted to support newest versions only, but I don't think it matters too much